.gitignore

```diff
@@ -16,3 +16,4 @@ __pycache__/
 .quilt_checked
 .source_dir
 ipkg-*/
+settings.json
```

Comment:
We don't add IDE related files to the `.gitignore`.
You can ignore this file in your global `.gitignore` file.

Reply:
done